docs/future-work/framework-improvements/automate-benchmark-experiments-analysis.md

```diff
@@ -1,3 +1,11 @@
 ---
 title: Automate Benchmark Experiments & Analysis
+description: 
```

Comment:
Just fyi this seems to be missing a description.

Reply:
I was going to ask @codeallthethingz to fill out this page :)